checkly/resource_trigger_group.go

```diff
@@ -71,58 +75,46 @@ func resourceTriggerGroupCreate(d *schema.ResourceData, client interface{}) erro
 		return fmt.Errorf("CreateTriggerGroup: API error: %w", err)
 	}
 
-	d.SetId(fmt.Sprintf("%d", result.ID))
-	d.Set("token", result.Token)
```

Comment:
is that ok to delete? It seems it is being used further down the line.

Reply:
It's being set by `resourceDataFromTriggerGroup` later

Reply:
It's being set by resourceDataFromTriggerGroup later